tests/core/test-core.c

```diff
@@ -830,6 +831,33 @@ static void test_brick_core_verify_re_link(void)
 	pg_brick_destroy(s);
 }
 
+static void test_brick_verify_re_link_monopole(void)
```

Comment:
You don't need to name it "verify" as a test always verify something.
Maybe you can name it "test_brick_monopole_unlink"